packages/cli/src/index.js

```diff
@@ -12,8 +12,10 @@ const program = require('commander');
 const generateCompilation = require('./lifecycles/compile');
 const runProdBuild = require('./tasks/build');
 const runDevServer = require('./tasks/develop');
+const ejectConfigFiles = require('./tasks/eject');
 const scriptPkg = require('../package.json');
-
+let compilation = {};
```

Comment:
oh, these were never declared before?  😳

Reply:
they were const and at the top of the switch condition